app/models/manageiq/providers/ibm_cloud/power_virtual_servers/network_manager/cloud_subnet.rb

```diff
@@ -1,5 +1,5 @@
 class ManageIQ::Providers::IbmCloud::PowerVirtualServers::NetworkManager::CloudSubnet < ::CloudSubnet
-  supports :create
+  supports_not :create
```

Comment:
This shouldn't be necessary, the default for `SupportsFeatureMixin` is `supports_not`